build/run-me.sh

```diff
@@ -72,6 +72,11 @@ MANDATORY parameters:
   --builddir PATH       Specify the root of the build tree.
 
 OPTIONAL parameters:
+  --boot-rot-key PATH   Path to the secure world root of trust private key
```

Comment:
If this is referring the TF-A root-of-trust key; it's an RSA private key, not a "private key certificate".

Reply:
Thanks for the clarification. https://github.com/ARMmbed/mbl-tools/pull/307/commits/8f369174a2748d07a644cb06ed0525d85d5b1f7f.